docs/fluent.md

```diff
@@ -0,0 +1,70 @@
+# Fluent API
+
+The Fluent API is the most basic component of the BDK. This component provides the developers very quickly an entry point 
```

Comment:
I don't think that the `Fluent API` is a "component"

Reply:
Should I call it "feature"?